Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -957,6 +957,26 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
 
         // expect this change stream to have more events after resuming
         expect(try changeStream2.nextWithTimeout()).toNot(beNil())
+
+        guard try client.serverVersion() >= ServerVersion(major: 4, minor: 2) else {
+            print("Skipping test case for server version \(try client.serverVersion())")
+            return
+        }
+
+        // test with startAfter
+        _ = try coll.renamed(to: self.getCollectionName(suffix: "2"))
+
+        _ = try changeStream2.nextWithTimeout()
+        expect(try changeStream2.nextWithTimeout()?.operationType).to(equal(.invalidate))
+
+        let opts = ChangeStreamOptions(startAfter: changeStream2.resumeToken)
+
+        // resuming (with startAfter) after an invalidate event should work
+        let changeStream3 = try coll.watch(options: opts)
+
+        try coll.insertOne(["kitty": "cat"])
```

Comment:
can we also add an insert before the call to `watch` and verify they're both seen? This tests the picking-up-where-you-left-off feature of `resumeAfter`.